storage/aptosdb/src/state_store/reset_lock.rs

```diff
@@ -0,0 +1,45 @@
+use super::buffered_state::BufferedState;
```

Comment:
This looks like #28 was merged here for some reason, was it due to some confusion on Slack?

I meant that to be reviewed and merged separately, it's not really needed for sync with upstream.